docs/pages/experiments/docs/blog.md

```diff
@@ -11,7 +11,7 @@ tags: ['Company']
 
 ### Image
 
-<img alt="Satellite image of Tenerife" src="/static/blog/2022-tenerife-retreat/tenerife.jpeg" style="aspect-ratio: 4/3;" loading="lazy" />
+<img alt="Satellite image of Tenerife" src="/static/blog/2022-tenerife-retreat/tenerife.jpeg" width="2560" height="1920" loading="lazy" />
```

Comment:
Not related to the other changes, but a quick win. This page is supposed to be the standard reference, here the standard reference to how to include images, this was outdated, and using the real width and height fix the layout shift.